README.md

```diff
@@ -19,24 +19,46 @@ The gDR suite offers a full stack solution for storing, processing, and visualiz
 			| BumpyMatrix |			                       ---------------------
 			+-------------+   \           +----------+                +-----------+
 		+----------------------+    --------  | gDRcore  | <------------- | gDRimport | 
-		| SummarizedExperiment |  /           +----------+   load raw     +-----------+
+		| MultiAssayExperiment |  /           +----------+   load raw     +-----------+
 		+----------------------+                   :        files for analysis
                                                            :
-				      perform              :   gDRcore::runDrugResponseProcessingPipeline()
+				      perform              :   gDR::runDrugResponseProcessingPipeline()
```

Comment:
It will not work in the current logic of the umbrella package. Once you do the `library(gDR)` you can either run `gDRcore::runDrugResponseProcessingPipeline()` or simply `runDrugResponseProcessingPipeline()`.

Reply:
Thanks for catching this. Where do we stand on [GDR-1905](https://jira.gene.com/jira/browse/GDR-1905)?

Reply:
We've frozen GDR-1905 due to other priorities. Maybe we can return to this in the next sprint (i.e. at the beginning of September)?

Reply:
Sounds good.